README.md

```diff
@@ -163,13 +162,56 @@ print("Response:", resp[0].choices[0].message.content)
 
 See [OpenAI Compatible API](https://predibase.github.io/lorax/guides/openai_api) for details.
 
+### Metrics
```

Comment:
Can you add this to docs instead? We have a section called Reference, we could add a page called Prometheus Metrics or similar.

Reply:
Also, is this something we can expose via the `openapi.json`?